### DISCUSSION TITLE
how to custom the name in the dashboard

### DISCUSSION THREAD

**Discussion by fbrliatr** [filamentphp/filament#13662] [Help] [answered]
### Package

Panel builder

### Package Version

v3.0.0

### How can we help you?

i want to custom the name below the welcome's logo. Can I know the easiest way to do?

**Comment by leandrocfe** [comment, marked as answer]
If you want to customize the view, you could do:

1 - Create a custom widget https://filamentphp.com/docs/3.x/panels/dashboard#custom-widgets

```php
class CustomAccountWidget extends Widget
{
    protected static ?int $sort = -3;
    
    protected static string $view = 'filament.widgets.custom-account-widget';
}
```

2 - Copy the original content, paste in your blade file and edit what you need

https://github.com/filamentphp/filament/blob/3.x/packages/panels/resources/views/widgets/account-widget.blade.php

3 - Use the new widget in your panel

```php
$panel
->widgets([
    //Widgets\AccountWidget::class,
    CustomAccountWidget::class,
    Widgets\FilamentInfoWidget::class,
])
```

**Reply by fbrliatr** [reply]
Thank you! I have another question, how can i make a content in the middle of table between the header and rows. like this one
![Screenshot 2024-07-22 142318](https://github.com/user-attachments/assets/7f3617dc-33fa-4f6a-84d1-49d5d4541456)

the 'tanggal angkut' and 'rute' part. i use table widget before. thank you in advance

**Reply by leandrocfe** [reply]
check render hooks
https://filamentphp.com/docs/3.x/support/render-hooks#overview
